backend/docker/prod/ecs/Dockerfile

```diff
@@ -48,26 +55,26 @@ RUN echo export PATH="\
     && /bin/bash -c  ". ~/.nvm/nvm.sh && cdk --version"
 
 # App specific requirements
-ADD backend/requirements.txt /requirements.txt
-ADD backend/dataall/base/cdkproxy/requirements.txt /cdk.requirements.txt
+ADD --chown=${CONTAINER_USER}:root backend/requirements.txt /requirements.txt
+ADD --chown=${CONTAINER_USER}:root backend/dataall/base/cdkproxy/requirements.txt /cdk.requirements.txt
 
 # Install App requirements
-RUN /bin/bash -c "${PYTHON_VERSION} -m pip install setuptools"
-RUN /bin/bash -c "${PYTHON_VERSION} -m pip install -r requirements.txt"
-RUN /bin/bash -c "${PYTHON_VERSION} -m pip install -r cdk.requirements.txt"
+RUN /bin/bash -c "${PYTHON_VERSION} -m pip install setuptools --prefix /usr/local"
+RUN /bin/bash -c "${PYTHON_VERSION} -m pip install -r requirements.txt --prefix /usr/local"
+RUN /bin/bash -c "${PYTHON_VERSION} -m pip install -r cdk.requirements.txt --prefix /usr/local"
 
 # App code
-ADD backend/dataall /dataall
+ADD --chown=${CONTAINER_USER}:root backend/dataall /dataall
 VOLUME ["/dataall"]
 ADD backend/cdkproxymain.py /cdkproxymain.py
```

Comment:
Why don't we change ownership in the line 62?

Reply:
Fixed @dlpzx